po/bn.po

```diff
@@ -56,38 +56,37 @@ msgstr "প্রথম দিনঃ সকাল"
 msgid "Welcome"
 msgstr "স্বাগতম"
 
-#: src/SUMMARY.md src/hello-world.md src/hello-world/hello-world.md
-#, fuzzy
+#: src/SUMMARY.md src/hello-world.md src/types-and-values/hello-world.md
 msgid "Hello, World"
-msgstr "Hello World!"
+msgstr "হ্যালো ওয়ার্ল্ড"
 
 #: src/SUMMARY.md src/hello-world/what-is-rust.md
 msgid "What is Rust?"
 msgstr "Rust কি?"
 
 #: src/SUMMARY.md src/hello-world/benefits.md
 msgid "Benefits of Rust"
-msgstr ""
+msgstr "Rust এর সুবিধা"
```

Comment:
```suggestion
msgstr "রাস্ট এর সুবিধা"
```
Let's not mix Bengali and English terms.

Reply:
> I saw the Japanese translation had kept "Rust" untranslated as a word which is why I was confused.

It's a really good question what we should do with names like "Rust" in languages that don't use Latin letters. I don't know myself what the custom is, so we've simply let it be up to the taste of the individual translators.

I would be interested in having a discussion about this, so feel free to raise the question in https://github.com/google/comprehensive-rust/discussions (if you want).

Reply:
I have started this discussion [here](https://github.com/google/comprehensive-rust/discussions/1888). Meanwhile, I am taking @mirwasi 's advice and translating it to Bengali with the same pronounciation. Would change it depending upon the consensus of the discussion.